docs/concepts/key-terms/tracing/distributed-tracing.mdx

```diff
@@ -243,4 +243,4 @@ Put simply: as a result of this head-based approach, where the decision is made
 
 ## Viewing Trace Data
 
-Through [Performance](/product/performance/) and [Discover](/product/discover-queries/), you can view trace data in the [event details](/product/sentry-basics/concepts/tracing/event-detail/).
+Through [Performance](/product/performance/) and [Discover](/product/discover-queries/), you can view trace data in the [trace view](/concepts/key-terms/tracing/trace-view/).
```

Comment:
We'll need to add the trace explorer page when it launches. Not sure if there are other entry points to the trace view, maybe also the insights modules and profiles?